hr_employee_firstname/models/hr_employee_public.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    # Fields coming from hr.employee.base
+    firstname = fields.Char(readonly=True)
```

Comment:
@astirpe ,

If you added a field on an hr.employee.base then no need to add on a public 
Please remove it

Reply:
On the public is set `readonly=True`. This is the way the employees V13 fields are made.